app/controllers/users.js

```diff
@@ -8,7 +8,10 @@ import mongoose from 'mongoose';
 import passport from 'passport';
 import avatarsList from './avatars';
 import { Tokenizer } from '../helpers/tokenizer';
-import sendInvitationEmail from '../helpers/sendInvitationEmail';
```

Comment:
Why the change of name here, is it necessary?

Reply:
Yes, I was getting an eslint error, that was the name of the method in the module he created